### DISCUSSION TITLE
[Github API] List comments for a pull request review github api - lack of status

### DISCUSSION THREAD

**Discussion by pmnoxx** [community/community#9175] [Pull Requests]
It looks like there is no way to find out which conversation is `resolved`/`unresolved` though the github api.

In the following `api`, there is no field, that states, whenever pull review comment has been resolved or not. Where can we find api for getting that field?
https://docs.github.com/en/rest/reference/pulls#list-comments-for-a-pull-request-review--code-samples

/repos/{owner}/{repo}/pulls/{pull_number}/reviews/{review_id}/comments'

**Comment by NicHaley** [comment]
+1

**Comment by maxmoehl** [comment]
FWIW this is available as part of the GraphQL API:
```graphql
type PullRequestReviewComment implements Comment & Deletable & Minimizable & Node & Reactable & RepositoryNode & Updatable & UpdatableComment {

  """
  Returns whether or not a comment has been minimized.
  """
  isMinimized: Boolean!

  """
  Returns why the comment was minimized. One of `abuse`, `off-topic`,
  `outdated`, `resolved`, `duplicate` and `spam`. Note that the case and
  formatting of these values differs from the inputs to the `MinimizeComment` mutation.
  """
  minimizedReason: String
}
```

But I would also prefer if it could be retrieved using the REST API.

**Comment by samot-gc** [comment]
For what it's worth, this is what I use:
```bash
gh api graphql -f query='{ repository(owner: "{owner}", name: "{repo}") { pullRequest(number: {pr_number}) { reviewThreads(first: 100) { nodes { isResolved comments(first: 100) { nodes { databaseId body path line author { login } } } } } } } }' --jq '.data.repository.pullRequest.reviewThreads.nodes[] | "Resolved: \(.isResolved)", (.comments.nodes[] | "- [\(.author.login)] comment_id=\(.databaseId) file=\(.path):\(.line)\n\(.body)"), "---"'
```
It finds each comment on a particular part of a file, and gets the corresponding conversation chain.

But I'd still prefer if it could be retrieved with the REST API too
